PelotonEppSdk/Interfaces/ICreditCardUpdateRequest.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using PelotonEppSdk.Models;
+
+namespace PelotonEppSdk.Interfaces
+{
+    public interface ICreditCardUpdateRequest
+    {
+        string OrderNumber { get; set; }
+        string CardOwner { get; set; }
+
+        CreditCardNumber CardNumber { get; set; }
+
+        string CardType { get; }
+        string ExpiryMonth { get; set; }
+        string ExpiryYear { get; set; }
+        string CardSecurityCode { get; set; }
```

Comment:
If we are not verifying the card on the update request thne we don't need CardSecurityCode to be sent